dbt/include/spark/macros/materializations/table.sql

```diff
@@ -98,7 +98,7 @@ else:
   msg = f"{type(df)} is not a supported type for dbt Python materialization"
   raise Exception(msg)
 
-df.write.mode("overwrite").format("delta").option("overwriteSchema", "true").saveAsTable("{{ target_relation }}")
```

Comment:
@benc-db any issues with  making this configurable for dbt-databricks?

Reply:
It already is: https://github.com/databricks/dbt-databricks/blob/e28ced5d32b8f2102284588a0dd293b142f792c6/dbt/include/databricks/macros/adapters/python.sql#L56